modules/install.js

```diff
@@ -38,14 +38,14 @@ export function install() {
       if (!cmdPromise) return null
 
       return cmdPromise
-        .then((actions) => {
```

Comment:
one issue with this is that errors in the action creators (not dispatching them, just creating them) would not generate a loop error, and I'd argue they should.

Maybe the solution is to generate the actions first, then catch, then dispatch after?